### ISSUE TITLE
conversation

### ISSUE THREAD

**Issue by tobyplowy** [DreamBlocks/DreamBlocks#2]
This is a place to Talk About different things like Graphics, ideas, improvements, or changes 
(whatever realy) just try to keep the offtopic minimal ok :)

**Comment by LordStephen77** [comment]
@tobyplowy Yes, it's a good idea. I am very busy with the work these days, now that I have created the organization and added you as official members you can handle it all quietly. Continue with the website, it's public and you need to set it up well. Thank you.

**Comment by AKryukov92** [comment]
I can't find a link to website you are talking about. Is it published in internet?

**Comment by LordStephen77** [comment]
@AKryukov92 https://dreamblocks.github.io/ Apparently github publishes it automatically by recognizing the repository name (mywebsite.github.io). There is a link to the organization's home page (top below the project title).

**Comment by tobyplowy** [comment]
@LordStephen77 You should make a development road map So contributers know what to work on so they Don't get in each others way also say what shuld be worked on first and what after that

**Comment by AKryukov92** [comment]
I agree with @tobyplowy. We need a backlog, filled with small tasks.
I'll add a number of simple tasks based on current codebase and slowly start to implement them.
But I will not add tasks about features. It is up to @LordStephen77 .

**Comment by LordStephen77** [comment]
it's a good idea. As soon as I get free from work I sort out the little wiki I'm writing, then the audience in the wiki of the project.

@AKryukov92 is enough to comment on the code you are writing. You can make any changes you like, feel free to work on this project as you believe it best. DreamBlocks is everyone, even yours!

**Comment by LordStephen77** [comment]
new chat gitter.im:

https://gitter.im/LordStephen77/Lobby

Here we can talk in our dedicated project space, so we keep ordering the official repository without the need to open 100 issues.

Thank you all.
